frontend/src/Pages/LoginSuccess.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { useSearchParams } from 'react-router-dom'
+import axios from 'axios';
+
+export default function LoginSuccess() {
```

Comment:
One thing that would be nice is some jsdoc that explains the purpose of each page, very simply.